src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -175,6 +175,62 @@ public static void Initialize()
             throw;
         }
 
+        try
+        {
+            Logger.Debug("Enabling by ref instrumentation.");
+            NativeMethods.EnableByRefInstrumentation();
+            Logger.Information("ByRef instrumentation enabled.");
+        }
+        catch (Exception ex)
+        {
+            Logger.Error(ex, "ByRef instrumentation cannot be enabled: ");
+        }
+
+        try
+        {
+            Logger.Debug("Enabling calltarget state by ref.");
+            NativeMethods.EnableCallTargetStateByRef();
+            Logger.Information("CallTarget State ByRef enabled.");
+        }
+        catch (Exception ex)
+        {
+            Logger.Error(ex, "CallTarget state ByRef cannot be enabled: ");
+        }
```

Comment:
Probably separate GH issue:
Can it be enabled by default? I do not see any condition when it should be disabled.